app/helpers/rooms_helper.rb

```diff
@@ -34,4 +34,8 @@ def calculated_menu_width(islet)
     nb_of_lanes = islet.bays.map(&:lane).max
     frame_width * (((nb_of_frames) / nb_of_lanes) + 1)
   end
+
+  def surface_area_with_suffix(surface_area)
+    "#{surface_area} #{t("surface_area.unit")}"
+  end
```

Comment:
Suggestion: since it's not full related to Room, maybe it could be move in a generic helper like `ApplicationHelper`?

Reply:
ok